README.md

```diff
@@ -1,9 +1,13 @@
 # clinical-trials
-Parsers for clinical trials data from clinicaltrials.gov
 
+Set of parsers and classes for clinical trials data from [ClinicalTrials.gov](https://clinicaltrials.gov)
 
 ## Installation
 
 ```bash
-$ pip install -e .  
-```
\ No newline at end of file
+pip install -e .  
+```
+
+## Usage 
```

Comment:
Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/WojciechKusa/clinical-trials/pullRequest?prid=9891375)
```suggestion
## Usage
```